### PR TITLE
Update of config file

### PR DESCRIPTION
**Pull request by rob-baron**
This does some minor reorganization to more generally handle all of the endpoints that need to be connected to in order to gather data.

### CODE REVIEW

**Review by tsachleben — COMMENTED** [review]
Appears dependent on #12 being pulled first. Please confirm.

**Comment by rob-baron** [comment]
Although it has more files in it than I expected, it does not require that #12 is done first.

#12 is dependent on this.

File: get_info.pl [hunk 1]
```diff
@@ -14,43 +14,22 @@
...
     }
 
 sub get_conn
     {
+    my $db_name=shift;
     my $user=shift;
     my $pass=shift;
 
-    my $conn = DBI->connect("dbi:Pg:dbname=postgres",$user,$pass);
+    print "-->> $db_name, $user, $pass \n";
```
Comment (kevinliang43) [inline get_info.pl]: Should we be printing the password?

Comment (tsachleben) [reply get_info.pl]: @robbaronbu we probably shouldn't be printing the password?

Comment (rob-baron) [reply get_info.pl]: good catch.

**Review by tsachleben — APPROVED** [review]

**Review by kevinliang43 — APPROVED** [review]

**Pull request merged**